data/releases/schedule.yaml

```diff
@@ -0,0 +1,67 @@
+schedules:
```

Comment:
How does this  file get generated/added/updated? It might be helpful to add a comment indicating that (lest people add entries to an autogenerated file)

Reply:
@celestehorgan -- I can fast-follow with details on this once the PR merges.

Reply:
I stole the file from [schedule-builder's readme](https://github.com/kubernetes/release/tree/master/cmd/schedule-builder). It looks like the file might be [manually](https://github.com/kubernetes/sig-release/commits/master/releases/schedule.yaml) updated (and same for patch-releases.md?) /cc @cpanato @xmudrii